executor/bin/prepare-os.sh

```diff
@@ -1,6 +1,6 @@
 #!/bin/sh
```

Comment:
How on earth is that Bourne shell with pipefail below...

```suggestion
#!/bin/bash
```

Reply:
Ok, there's a lot of it